database/index.js

```diff
@@ -107,6 +107,37 @@ const Comments = sequelize.define('comment', {
   },
 });
 
+const Messages = sequelize.define('messages', {
+  // convoID: {
+  //   type: Sequelize.INTEGER,
+  //   references: { model: 'convos', key: 'id' },
+  // },
+  incoming: {
+    type: Sequelize.BOOLEAN,
+  },
+  body: {
+    type: Sequelize.STRING,
```

Comment:
I think the default for this is 255 char, it might have to be a little long for a text message, maybe 500